### ISSUE TITLE
Launch pipeline regex fails

### ISSUE THREAD

**Issue by Emiller88** [nf-core/nf-co.re#828]
**Describe the bug**
When trying to launch a new workflow some of the regex check fail
![image](https://user-images.githubusercontent.com/20095261/126085926-e940ac04-21cc-4a48-a7ce-855ffdc2e4c9.png)
But `.csv` Passes

**To Reproduce**
1. Go to Launch Pipeline for rnaseq
2. Add `samplesheet.csv`
3. Scroll down to launch

**Expected behavior**
The regex passes because it ends in csv

**Screenshots**
![image](https://user-images.githubusercontent.com/20095261/126085926-e940ac04-21cc-4a48-a7ce-855ffdc2e4c9.png)

**Browser (please complete the following information):**
 - OS: NixOS 21.05
 - Browser: Firefox
 - Version 88.0.1

**Comment by Emiller88** [comment]
I have a feeling it's either in https://github.com/nf-core/rnaseq/blob/7919c2db266d9f470509fc19e569112207d0cde9/nextflow_schema.json#L18
Or in the way the site parses it. It also seems to fail with anything that starts with `\\.` so the gtf or .gtf.gz for the gtf file also fails similarly.

**Comment by mashehu** [comment]
Good catch! The problem is that the HTML form field wants the regex to match the whole string, therefore the pattern in the `nextflow_schema.json` needs to be updated. I guess `\S*\.csv$` should be sufficient. Moving the issue to the RNAseq repo.
